.circleci/config.yml

```diff
@@ -427,6 +426,53 @@ jobs:
       - store_artifacts:
           path: /tmp/ds054
 
+  build_docs:
+    docker:
+      - image: python:latest
+    environment:
+      - FSLOUTPUTTYPE: NIFTI
+    steps:
+      - checkout
+      - run:
+          name: Install Graphviz
+          command: apt update && apt -y install graphviz
+      - run:
+          name: Install deps
+          command: pip install -r requirements.txt && pip install -r docs/requirements.txt
+      - run:
+          name: Build only this commit
+          command: sphinx-build -W ./docs ./docs/_build/no_version_html
+      - run:
+          name: Generate Docs
+          command: sphinx-versioning -vv build -r ci_doc_deploy ./docs ./docs/_build/html/
```

Comment:
I don't see the Makefile used anywhere here? Shouldn't we have only one way of generating docs? (I'd like the `make`, btw)